A/accerionsensorapi/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "accerionsensorapi"
```

Comment:
Maybe call it
```suggestion
name = "AccerionSensorAPI"
```
like the upstream package?  Note that if you change it you'll need to change also the name the directory to exactly match the new name